api/openapi.yaml

```diff
@@ -74,22 +74,34 @@ paths:
       responses:
         200:
           description: successful operation
-          headers:
-            X-Total-Count:
-              schema:
-                type: integer
-              description: Total records count
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "./transactions.yaml#/components/schemas/Transaction"
+                type: object
+                properties:
+                  count:
+                    type: integer
+                  items:
+                    type: array
+                    items:
+                      $ref: "./transactions.yaml#/components/schemas/Transaction"
```

Comment:
Should we update this Transaction def to include the `type` and `spec` format?

Reply:
+1

Reply:
oh yeah, forgot we don't have those fields in this object. Fixing that!